gifify.sh

```diff
@@ -82,13 +97,24 @@ fi
 fps=$(echo $fpsspeed | cut -d'@' -f1)
 speed=$(echo $fpsspeed | cut -d'@' -f2)
 
-if [ ! -z "$speed" ]; then
+if [ -z "$speed" ]; then
   speed=1
 fi
 
+if [ ! -z "$to" ]; then
```

Comment:
You'll need to add the `to=` and `from=` vars, otherwise you'll get:
```
/usr/local/bin/gifify: line 104: to: unbound variable
/usr/local/bin/gifify: line 104: temp: unbound variable
```